crates/dbsp/README.md

```diff
@@ -0,0 +1,116 @@
+# DBSP
+
+DBSP is a computational engine for continuous analysis of changing
+data.  With DBSP, a programmer writes code in terms of computations on
+a complete data set, but DBSP implements it incrementally, meaning
+that changes to the data set run in time proportional to the size of
+the change rather than the size of the data set.  This is a major
+advantage for applications that work with large data sets that change
+frequently in small ways.
+
+The DBSP computational engine is a component of the [Feldera
+Continuous Analytics Platform](https://www.feldera.com).
+
+# Resources
+
+## Learning Materials
+
+- The [tutorial] walks through a series of simple examples.
+
+- The [`circuit` module documentation] provides a catalog of
+  computational elements.
+
+- More sophisticated uses can be found as [examples] and [benchmarks]
```

Comment:
We probably also want a link to the paper.

Reply:
Added.